### PR TITLE
Reviews

### PR DESCRIPTION
**Pull request by xnick421**

### CODE REVIEW

File: app/controllers/review_controller.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  end
+
+  def edit
+    @review = Review.find(params[:id])
+  end
+
+  def update
+    @review = Review.find(params[:id])
+    if @review.update_attributes(review_params)
+      # redirect and flash message
```
Comment (justinhuynh) [inline app/controllers/review_controller.rb]: Ditch unused commented code - or finish writing the `redirect` and `flash` message

File: app/controllers/review_controller.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      # redirect
+    else
+      # flash alert and render
+    end
+  end
+
+  def destroy
+    @review = Review.find(params[:id])
+    if @review.destroy!
+      # flash[:notice] = "Review deleted."
```
Comment (justinhuynh) [inline app/controllers/review_controller.rb]: Ditch the unused code

File: app/models/review.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class Review < ActiveRecord::Base
+  belongs_to :bar
+  belongs_to :user
+
+  validates :title, presence: true
+  validates :body, presence: true
+  validates :bar_id, presence: true
+  validates :user_id, presence: true
+  validates :rating, presence: true,
+  numericality: { integer: true }, inclusion: { in: 1..5 }
```
Comment (cmkoller) [inline app/models/review.rb]: I'd say the norm is for this should read something like:

```
validates :rating, 
  presence: true,
  numericality: { integer: true }, 
  inclusion: { in: 1..5 }
```

(Pretty sure that's what Hound is looking for!)

File: app/controllers/review_controller.rb [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  def new
+    @review = Review.new
+  end
+
+  def create
+    @bar = Bar.find(params[:bar_id])
+    @review = Review.new(review_params)
+    @review.bar = @bar
+    @review.user = current_user
+    Binding.pry
```
Comment (cmkoller) [inline app/controllers/review_controller.rb:34]: You've got a `binding.pry` in here!

File: app/controllers/review_controller.rb [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    end
+  end
+
+  def destroy
+    @review = Review.find(params[:id])
+    if @review.destroy!
+      flash[:notice] = "Review deleted."
+      redirect_to bars_path
+    else
+      lash[:notice] = "Bar did not delete."
```
Comment (cmkoller) [inline app/controllers/review_controller.rb:48]: I think this might be a typo? I'm assuming you mean `flash`

**Pull request closed without merging**